src/lib/rest/StringFilter.cpp

```diff
@@ -497,7 +522,7 @@ bool StringFilterItem::parse(char* qItem, std::string* errorStringP)
   }
   if (forbiddenChars(lhs, ""))
   {
-    *errorStringP = "invalid character found in URI param /q/";
+    *errorStringP = "invalid character found in URI param /q/";  // fix this (and others)
```

Comment:
I don't remember where this comment come from... Maybe in a early stage of the work, when I wrongly thought that check could be a problem.

However, make test ensures that all works fine, so I have removed the comment.

Fixed in 5ac5ef1